packages/react-cosmos/src/components/proxies/PropsProxy.jsx

```diff
@@ -10,11 +10,15 @@ export default class PropsProxy extends React.Component {
       fixture,
       onComponentRef,
     } = this.props;
+    let children;
+    if (Array.isArray(fixture.children)) {
+      children = [].slice.call(fixture.children);
+    }
 
     return React.createElement(component, {
       ...fixture,
```

Comment:
The reason this code exists is because `createElement` takes a variable number of arguments, but `children` is an `Array`. Ideally you'd destructure it with `...children`, but this fails if the fixture does not have the `children` field (you cannot destructure `undefined`).